src/main/SearchForm.js

```diff
@@ -0,0 +1,121 @@
+import React from "react";
+import { useState } from "react/cjs/react.development";
+import "./SearchForm.css";
+
+const beersUrl = `https://api.punkapi.com/v2/beers`;
+
+function SearchForm(props) {
+  const [inputText, setInputText] = useState("");
+  const [show, setShow] = useState(false);
+  const [alcoholVol, setAlcoholVol] = useState(2);
+  const [bitterness, setBitterness] = useState(0);
+  const [colorEBC, setColorEBC] = useState(4);
+  const searchUrl = `${beersUrl}?beer_name=${inputText}`;
+  let searchList;
+  let setBeerList = props.setBeerList;
```

Comment:
You can use destructuring here. Will be easier to read and write. `const { setBeerList } = props;`

OR you can simply use destructuring in the function parameters. `function SearchForm({ setBeerList })`